package-lock.json

```diff
@@ -4069,6 +4070,795 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
+    "fsevents": {
```

Comment:
A rebase / rerun from within your container should be run to ensure this has been removed before merging.

Reply:
Thank you for catching that. I'll try to do that now.

Reply:
Rebasing didn't remove it. What do you mean by "rerun from within your container"? `npm install` from within the container shell? I tried that too... :(

Reply:
I was able to remove this, can you try the following? If it doesn't work I can just push that change up (but that seems like a super not-scalable flow going forward, so hopefully we can figure this out)

- (on local host machine, within edx-platform directory): `rm -rf package-lock.json`
- (on devstack, immediately after `make lms-shell`): `rm -rf node_modules/`
  - this step got me a `rm: cannot remove 'node_modules/': Device or resource busy` error, so it may not actually be needed?
- (same devstack session) `npm install`
- (back on host machine's edx-platform checkout) `git diff`, see that package-lock is updated.

Reply:
I gave that a shot. It did change package-lock, but it also had fsevents installed. I'm assuming we don't want that, so I'll check back in in the morning.